src/web/tools/KeyEventManager.ts

```diff
@@ -0,0 +1,81 @@
+import { AdaptedEvent, EventTypes } from '../interfaces';
+import EventManager from './EventManager';
+import { PointerType } from '../../PointerType';
+
+export default class KeyEventManager extends EventManager<HTMLElement> {
+  private activationKeys = ['Enter', ' '];
+  private cancelationKeys = ['Tab'];
+  private isPressed = false;
+
+  private keyDownCallback = (event: KeyboardEvent): void => {
+    if (this.cancelationKeys.indexOf(event.key) !== -1 && this.isPressed) {
+      this.dispatchEvent(event, EventTypes.CANCEL);
+      return;
+    }
+
+    if (this.activationKeys.indexOf(event.key) === -1) {
+      return;
+    }
+
+    this.dispatchEvent(event, EventTypes.DOWN);
+  };
+
+  private keyUpCallback = (event: KeyboardEvent): void => {
+    if (this.activationKeys.indexOf(event.key) === -1 || !this.isPressed) {
+      return;
+    }
+
+    this.dispatchEvent(event, EventTypes.UP);
+  };
+
+  private dispatchEvent(event: KeyboardEvent, eventType: EventTypes) {
+    const adaptedEvent = this.mapEvent(event, eventType);
+    switch (eventType) {
```

Comment:
```suggestion
    const adaptedEvent = this.mapEvent(event, eventType);
    
    switch (eventType) {
```

Reply:
Fixed in [b76a0ef](https://github.com/software-mansion/react-native-gesture-handler/pull/3035/commits/b76a0ef94344932b1669b2dc7b0f5b6d5a813279)